pkg/library/lifecycle/prestart_test.go

```diff
@@ -56,6 +56,9 @@ func TestGetInitContainers(t *testing.T) {
 		loadPreStartTestCaseOrPanic(t, "prestart_exec_command.yaml"),
 		loadPreStartTestCaseOrPanic(t, "prestart_apply_command.yaml"),
 		loadPreStartTestCaseOrPanic(t, "init_and_main_container.yaml"),
+		loadPreStartTestCaseOrPanic(t, "persistent_home_initcontainer_first_initcontainer.yaml"),
+		loadPreStartTestCaseOrPanic(t, "persistent_home_initcontainer_second_initcontainer.yaml"),
+		loadPreStartTestCaseOrPanic(t, "persistent_home_initcontainer_first_initcontainer.yaml"),
```

Comment:
I think you intended to load `persistent_home_initcontainer_only_initcontainer.yaml` here instead, as `persistent_home_initcontainer_first_initcontainer.yaml` was already loaded.